tests/src/FunctionalJavascript/CorporateContactFormTest.php

```diff
@@ -615,6 +705,34 @@ protected function checkCorporateFieldsInStorage($max_delta = 1): void {
       $value = $contact_form->getThirdPartySetting('oe_contact_forms', $key);
       $this->assertEquals($expected, $value);
     }
+
+    // Check saved language options.
+    $value = $contact_form->getThirdPartySetting('oe_contact_forms', 'override_languages');
+    if ($language_options_filled) {
+      $expected = [
+        0 => 'http://publications.europa.eu/resource/authority/language/SWE',
+        1 => 'http://publications.europa.eu/resource/authority/language/SPA',
+        2 => 'http://publications.europa.eu/resource/authority/language/CES',
+        22 => 'http://publications.europa.eu/resource/authority/language/FIN',
+        23 => 'http://publications.europa.eu/resource/authority/language/BUL',
+      ];
```

Comment:
Same here, use a constant and assert all.

Reply:
answered ^^